### DISCUSSION TITLE
Need help to replace the column name of Move, Tooltip content change and css for filter input and clear icon

### DISCUSSION THREAD

**Discussion by nowshads** [KevinVandy/material-react-table#720] [Q&A] [answered]
1. How can i remove move column name from the headers
2. How can i change the colour of text in the filter inputs and clear icon 
3. How can i change the tooltip of move(drag) icon .

**Comment by KevinVandy** [comment, marked as answer]
Read the display of column docs and the localization docs

**Reply by nowshads** [reply]
is these ? https://www.material-react-table.com/docs/guides/localization

**Reply by nowshads** [reply]
Found the solution Thank you :)

**Comment by itshallrun** [comment]
@nowshads what is the solution? Share it with us please

**Reply by KevinVandy** [reply]
Read the docs

**Reply by itshallrun** [reply]
The solution is:
on the table options set
```
    displayColumnDefOptions: {
      "mrt-row-drag": {
        header: "",
      },
    },
```
